tests/test_logs.py

```diff
@@ -96,45 +95,41 @@ def assert_file_log_level(logs_path: Path, expected_level: str) -> None:
     assert_log_level(_get_logs_file(logs_path).read_text(), expected_level)
 
 
-def assert_log_dir_is_empty(logs_path: Path) -> None:
-    assert len(list(logs_path.iterdir())) == 0
+def assert_log_is_empty(logs_path: Path) -> None:
+    with open(logs_path / "snowcli.log", "r") as log:
+        log_content = log.read()
+    assert log_content == ""
```

Comment:
```suggestion
    assert _get_logs_file(logs_path).read_text() == ""
```
To be consistent with the rest of the file :)

Reply:
Changed